shell/common/rasterizer.cc

```diff
@@ -68,6 +68,17 @@ void Rasterizer::Teardown() {
   last_layer_tree_.reset();
 }
 
+void Rasterizer::PurgeCaches() const {
```

Comment:
Here and elsewhere, let's call this `NotifyLowMemoryWarning`

Reply:
Done